bindings/postgres/metadata.go

```diff
@@ -14,26 +14,41 @@ limitations under the License.
 package postgres
 
 import (
+	"time"
+
 	pgauth "github.com/dapr/components-contrib/common/authentication/postgresql"
+	awsiam "github.com/dapr/components-contrib/common/component/postgresql/awsIAM"
 	kitmd "github.com/dapr/kit/metadata"
 )
 
+const (
+	defaultTimeout = 20 * time.Second // Default timeout for network requests
+)
+
 type psqlMetadata struct {
 	pgauth.PostgresAuthMetadata `mapstructure:",squash"`
+	awsiam.AWSIAM               `mapstructure:",squash"`
```

Comment:
Just FYI - by including this here each included metadata (which is merged into this struct) must be included in the `metadata.yaml` file. There is a CI task that will validate this.
You may have already noticed this because you updated the metadata.yaml - so no action on that here.